zenodo/modules/records/serializers/__init__.py

```diff
@@ -77,11 +78,18 @@
 #: DataCite serializer
 datacite_v31 = ZenodoDataCite31Serializer(DataCiteSchemaV1, replace_refs=True)
 datacite_v41 = ZenodoDataCite41Serializer(DataCiteSchemaV4, replace_refs=True)
+#: DCAT serializer
+dcat_v41 = DcatSerializer()
```

Comment:
Is DCAT's version `4.1`, i.e. does it match DataCite's? If not, `dcat_v1` should be fine